### ISSUE TITLE
Old CMS: Bug in Event Accessibility Note

### ISSUE THREAD

**Issue by saumier** [culturecreates/footlight-admin#176]
Cannot save event in unilingual calendar (see Signé Laval) due to bug in Event Accessibility Note.  Temporary patch released to enable Signé Laval to create test events so they can develop their website.

https://github.com/culturecreates/footlight-admin/commit/0c84e951194b9e5a66b38a3c5c5bb89eea160d52

**Comment by saumier** [comment]
@raseemayatt I cannot save the Event Accessibility Note. When I return to the event after entering an Event Accessibility Note it does not appear on the screen.
